docusaurus/docs/guides/deployments/10-linux/20-router/40-cli-mgmt.mdx

```diff
@@ -6,11 +6,10 @@ sidebar_label: CLI Mgmt
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
-import CliLogin from '/docs/_cli-login.md'
 
 ## Login
 
-<CliLogin/>
+Enroll an admin identity and log in. [Link to instructions](/reference/40-command-line/login.mdx)
```

Comment:
I don't like the "Enroll an admin identity and log in." wording. Let's be more generic.

> To use the CLI, you'll need to be logged in. [Link to instructions]()